jbdevstudio/com.jboss.jbds.central.discovery.earlyaccess/plugin.xml

```diff
@@ -75,5 +65,30 @@ This content is unavailable. Enable Early Access to see it.</description>
          <overview url="http://www.jboss.org/tools"/>
       </connectorDescriptor>
 
+       <!-- JBoss Fuse Development -->
```

Comment:
Because the IS content isn't easily integrated into our snapshot site yet. And I can push a PR into our code, rebuild it, and have a site in a few mins w/o having to coordinate w/ Paul to get a build done. Also doesn't require commandline flags or changes to ide-config.props.